clang/include/clang/Basic/AttrDocs.td

```diff
@@ -6826,6 +6826,19 @@ def YkUnrollSafeDocs : Documentation {
   }];
 }
 
+def YkPromoteDocs : Documentation {
+  let Category = DocCatFunction;
+  let Content = [{
+  The ``yk_promote`` attribute tells the Yk JIT which named function arguments
```

Comment:
Wait, I see that can't happen, because we need to specify using the variable name which ones we want to promote. Actually, my whole question doesn't make any sense, since you can't have constants in function definitions. Just forget what I said. :laughing:

Reply:
We raced, but yeah :)